pyfar/io/io.py

```diff
@@ -461,6 +463,320 @@ def default_audio_subtype(format):
     return soundfile.default_subtype(format)
 
 
+def read_comsol(filename, expressions=None, parameters=None):
+    """Read exported data from COMSOL.
+
+    Parameters
+    ----------
+    filename : str, Path
+        Input file. Excepted input files are '.txt', '.dat' and '.csv'. '.csv'
+        Files are strongly recommended, since .txt and .dat are inconvenient
+        in their format.
+    expressions : list[str], optional
+        This list of expressions defines the output expressions. An example
```

Comment:
- This is hard to understand for a non COMSOL user. Should we give at least a brief description of what expressions are in COMSOL?

- I would change the term 'defines the output expressions' because we are not returning expressions. How about 'defines which parts of the COMSOL data is returned as a pyfar audio object'?

- The links should start with pyfar.io but start with pyfar.dsp

- The link to `read_comsol_header` did not really clarify fro me. How about writing 'a list of all expressions can be obtained with ...'?